### PR TITLE
Div 3893 use govuk frontend

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-4137 - Update to GOV.UK Frontend](https://tools.hmcts.net/jira/browse/DIV-4137)
 - Update AOS to use the latest gov.uk frontend

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,15 +18,14 @@
     "express": "^4.15.3",
     "express-nunjucks": "^2.2.3",
     "extract-text-webpack-plugin": "^3.0.0",
-    "govuk-elements-sass": "^3.1.1",
+    "govuk-frontend": "^2.6.0",
     "govuk_frontend_toolkit": "^8.1.0",
```
Comment (hosainnet) [inline package.json]: `govuk_frontend_toolkit` should be removed

File: assets/scss/look-and-feel/gov-uk-frontend-toolkit-backward-compatable.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* Hide for both screenreaders and browsers */
```
Comment (hosainnet) [inline assets/scss/look-and-feel/gov-uk-frontend-toolkit-backward-compatable.scss]: Sorry minor typo in the file name `compatible` - might have to update references in DN and RFE though

File: examples/test-app/assets/scss/main.scss [hunk 3]
```diff
@@ -33,52 +33,26 @@
...
     margin-bottom: 20px;
   }
 
   // Fix grid layout within example boxes for IE7 and below
   // where box-sizing isn't supported: http://caniuse.com/#search=box-sizing
   @mixin example-box-column($width) {
-    width: (($site-width - $gutter) * $width) - $gutter;
-  }
-
-  @include ie-lte(7){
```
Comment (hosainnet) [inline examples/test-app/assets/scss/main.scss]: IE 7 lol

File: templates/look-and-feel/components/fields.old.njk [hunk 4]
```diff
@@ -0,0 +1,368 @@
+{#
```
Comment (hosainnet) [inline templates/look-and-feel/components/fields.old.njk]: Why do we need this `fields.old.njk` file?

**Comment by dtaylor7** [comment]
@hosainnet 👍

**Pull request merged**